src/components/plots/piePlot.ts

```diff
@@ -74,11 +74,13 @@ export module Plots {
       var defaultFillFunction = (d: any, i: number) => this._colorScale.scale(String(i));
       attrToProjector["fill"] = attrToProjector["fill"] || defaultFillFunction;
 
+      this._propertyBindings.forEach((key, binding) => attrToProjector[key] = Pie._scaledAccessor(binding));
```

Comment:
Properly, the property-bindings should go on before the attr-bindings, so that `attr()` takes precedence over the property setters. We can emulate that functionality here by checking if `attrToProjector[key]` already exists and not setting the property in that case.